src/ServiceManager/Travely.ServiceManager.Service/Travely.ServiceManager.Service.csproj

```diff
@@ -5,11 +5,17 @@
   </PropertyGroup>
 
```

Comment:
Please consider to set AssemblyName & RootNamespace

Reply:
done